torch/_dynamo/_trace_wrapped_higher_order_op.py

```diff
@@ -106,9 +106,8 @@ def _trace_wrapped_op_dense(*args, fn):
 @_trace_wrapped_op.py_functionalize_impl
 def _trace_wrapped_functionalized(ctx, *args, fn):
     unwrapped_args = ctx.unwrap_tensors(args)
-    wrapped_fn = ctx.functionalize(fn)
     with ctx.redispatch_to_next():
-        return ctx.wrap_tensors(_trace_wrapped_op(*unwrapped_args, fn=wrapped_fn))
+        return ctx.wrap_tensors(_trace_wrapped_op(*unwrapped_args, fn=fn))
```

Comment:
move this up the stack?

Reply:
Yeah, oops.

Reply:
Hmm that PR landed, I shall keep this here.